### ISSUE TITLE
Plan for WebDriver BiDi support in testdriver?

### ISSUE THREAD

**Issue by rakuco** [web-platform-tests/wpt#45404]
I've tried looking for an existing issue but couldn't find any. Are there any plans to add support for BiDi to testdriver, or a tracking bug/project?

In particular, [the BiDi version of SetPermission](https://w3c.github.io/permissions/#webdriver-bidi-command-permissions-setPermission) allows specifying an origin, which help with several cross-origin tests that do not work as expected today.

**Comment by rakuco** [comment]
cc @jgraham @OrKoN

**Comment by OrKoN** [comment]
@rakuco yes, the use of WebDriver BiDi is pending on https://github.com/web-platform-tests/rfcs/pull/185

**Comment by foolip** [comment]
https://github.com/web-platform-tests/rfcs/pull/185 was merged about a month ago and there's an implementation in progress in https://github.com/web-platform-tests/wpt/pull/45823.
